testsuite/MDAnalysisTests/topology/test_txyz.py

```diff
@@ -30,7 +30,7 @@
 class TestTXYZParser(ParserBase):
     parser = mda.topology.TXYZParser.TXYZParser
     guessed_attrs = ['masses']
-    expected_attrs = ['ids', 'names', 'bonds', 'types']
+    expected_attrs = ['ids', 'names', 'bonds', 'types', 'elements']
```

Comment:
If you follow the suggestion I made above, you can use the PDB parser as an example for the tests:

https://github.com/MDAnalysis/mdanalysis/blob/e7ee5a4a077c2179d6037ba634240ce7ebcb76ef/testsuite/MDAnalysisTests/topology/test_pdb.py#L266

https://github.com/MDAnalysis/mdanalysis/blob/e7ee5a4a077c2179d6037ba634240ce7ebcb76ef/testsuite/MDAnalysisTests/topology/test_pdb.py#L278

https://github.com/MDAnalysis/mdanalysis/blob/e7ee5a4a077c2179d6037ba634240ce7ebcb76ef/testsuite/MDAnalysisTests/topology/test_pdb.py#L308